openvdb/tools/FindActiveValues.h

```diff
@@ -0,0 +1,410 @@
+///////////////////////////////////////////////////////////////////////////
+//
+// Copyright (c) Ken Museth
+//
+// All rights reserved. This software is distributed under the
+// Mozilla Public License 2.0 ( http://www.mozilla.org/MPL/2.0/ )
+//
+// Redistributions of source code must retain the above copyright
+// and license notice and the following restrictions and disclaimer.
+//
+// *     Neither the name of DreamWorks Animation nor the names of
```

Comment:
Think you want to drop this Dreamworks section?

Reply:
good point :)

Reply:
@danrbailey, can you confirm that for new files, we agreed to assign the copy rights to ourselves and more to the point not to Dreamworks?

Reply:
Yeah, that's what I think we agreed, I'm using a Lucasfilm copyright.